.annotation_safe_list.yml

```diff
@@ -0,0 +1,26 @@
+# This is a Code Annotations automatically-generated Django model safelist file.
+# These models must be annotated as follows in order to be counted in the coverage report.
+# See https://code-annotations.readthedocs.io/en/latest/safelist.html for more information.
+#
+# fake_app_1.FakeModelName:
+#    ".. no_pii::": "This model has no PII"
+# fake_app_2.FakeModel2:
+#    ".. choice_annotation::": foo, bar, bazadmin.LogEntry: {}
```

Comment:
This fell afoul of the same bug with safelist generation as Julia's. I'll work on a fix for that tomorrow, but it will need a newline added.

Reply:
I'm actually going to delete admin.LogEntry: {} because it is a 3rd party model which doesn't need to be annotated in this non-IDA repository.